MIP/mip-74/README.md

```diff
@@ -0,0 +1,159 @@
+
+![alt text](overview.png)
+_Figure 1: Architecture of the Rate Limitation_
+
+### Actors, Components and Trust assumptions
+
+We assume the following trust assumptions:
+
+1. The Governance contract is implemented correctly.
+1. The Bridge contract and the L2 Native Bridge contract are implemented correctly.
+1. The Governance Operator is trusted. For example it COULD be a multisig human.
+1. The Relayer is a trusted software component.
+
+### Risks and mitigation strategies
+
+The following risks are associated with the Native Bridge:
+
+1. The trusted relayer is compromised or faulty. We thus want to ensure that the relayer has not unlimited power to release or mint assets. For this we MUST implement a rate limiter on the target chain.
+1. In order to rate limit the bridge (e.g. stop the bridge transfers entirely) there should be a higher instance than the relayer in setting rate limits. Thus the rate limit on the target chain SHOULD be set by the Operator.
+1. The Relayer may go down, while the number of transactions and requested transfer value across the bridge still increases on the source chain. Due to rate limit on the target chain the Relayer may struggle to process all initiated transfers. Thus the Relayer or the Operator MUST rate limit the source chain as well.
+
+### Objectives
+
+The objectives of the rate-limiter are to guarantee the following properties:
+
+- the value of assets being bridged MUST always be less than the insurance funds
+- property 2
+- property 3 ...
+
+### Rate-Limiter
+
+In the Native Bridge, the Rate-Limiter MUST be implemented as part of the L1 Native Bridge contract and the L2 Native Bridge contract.
+
+#### Insurance funds
+
+ We assume there is a Insurance Fund on both L1 and L2, with values `insurance_fund_L1` and `insurance_fund_L2`, respectively.
+ These values are considered constant in the sequel. There may be updated if needed or if new funds are added to the pools.
+
+The Insurance Fund rate-limits the outbound transfers, i.e. for a given transfer from source chain to target chain the Insurance Fund on the target chain is responsible for the rate limit, and thus we will refer to the `insurance_fund_target`. I.e. for a transfer from L1 to L2 the `insurance_fund_target = insurance_fund_L2` is responsible for the rate limit. While for a transfer from L2 to L1 the `insurance_fund_target = insurance_fund_L1` is responsible for the rate limit.
+
+#### Rate limit on the target chain
+
+The rate limit is dependent on the fund size in the Insurance Fund. In particular the maximum rate limit
+
+`max_rate_limit_target = insurance_fund_target / reaction_time`,
+
+where the `reaction_time` is the time it takes for the Operator to react to a faulty or compromised component. The `reaction_time` is a parameter that is set by the Operator. The Operator MAY set the actual rate limit lower than the `max_rate_limit_target`. However the Rate Limiter MUST NOT set the rate limit higher than the `max_rate_limit_target`.
```

Comment:
@apenzk
I suggest to break this into two independent parts:
- property if the rate limit and max_rate limit etc
- permissions: who can set what.

At the moment I find this section confusing with the Rate limier, the Operator etc so it may need to be clarified.